tests/test_flow360_params.py

```diff
@@ -176,7 +176,9 @@ def test_flow360param():
 
 def test_flow360param1():
     with flow360.SI_unit_system:
-        params = Flow360Params(freestream=FreestreamFromVelocity(velocity=10 * u.m / u.s))
+        params = Flow360Params(
+            freestream=FreestreamFromVelocity(velocity=10 * u.m / u.s), boundaries={}
```

Comment:
Does the user have to add the empty boundaries every time now? If so, shouldn't we handle this via some default?

Reply:
the boundaries are obligatory for every case and we need explicitly set them. Implicit settings can lead to unexpected behaviour